### PR TITLE
Added option to disable Kerberos FAST

### PR DESCRIPTION
**Pull request by AlexanderRasmussen**
Got the same issue with kowl as this issue in [kminion #111](https://github.com/cloudhut/kminion/issues/111) so I basically just copied the implementation that @weeco did there.

I have built the docker file and tested it locally and it fixed the problem.

### CODE REVIEW

File: docs/config/kowl-business.yaml [hunk 1]
```diff
@@ -44,6 +44,7 @@ kafka:
   #     username:
   #     password: # can be set via the --kafka.sasl.gssapi.password flag as well
   #     realm:
+  #     enableFast: True
```
Comment (weeco) [inline docs/config/kowl-business.yaml:47]:
```suggestion
  #     enableFast: true
```

Just for consistency :)

File: docs/config/kowl.yaml [hunk 2]
```diff
@@ -44,6 +44,7 @@ kafka:
   #     username:
   #     password: # can be set via the --kafka.sasl.gssapi.password flag as well
   #     realm:
+  #     enableFast: True
```
Comment (weeco) [inline docs/config/kowl.yaml:47]:
```suggestion
  #     enableFast: true
```

**Review by weeco — COMMENTED** [review]
Thanks, I wanted to add that to Kowl anyways!

**Comment by AlexanderRasmussen** [comment]
Should have fixed your comments now :)

**Review by weeco — APPROVED** [review]
Thank you!

**Pull request merged**